src/pages/explore/[[...slug]].js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+import ExplorePage from "@/pesayetu/components/ExplorePage";
+import Page from "@/pesayetu/components/Page";
+import { tileLayers } from "@/pesayetu/config";
+import getPostTypeStaticProps from "@/pesayetu/functions/postTypes/getPostTypeStaticProps";
+import fetcher from "@/pesayetu/utils/fetcher";
+
+export default function Explore(props) {
+  return (
+    <Page {...props}>
+      <ExplorePage {...props} />
+    </Page>
+  );
+}
+
+export async function getStaticPaths() {
+  const result = await fetcher(
+    `${process.env.HURUMAP_API_URL}all_details/profile/1/geography/KE/?format=json`
+  );
+  const featuredCountiesCode =
+    process.env.NEXT_PUBLIC_FEATURED_COUNTIES.split(",");
```

Comment:
1. Shouldn't `featuredCountiesCode` be based to Layers and used [here](https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup)
2. Is this something that is fixed-ish or can change often (ENV vs CMS).

Reply:
@kilemensi confused with the above comments

Reply:
🤯 ... I meant:

1. We're already loading featuredCountiesCode here and repeating the same thing [here](https://github.com/CodeForAfrica/PesaYetu/pull/89/files#diff-3f05ca11f5b46d1d0d495c906f5b0d0cdd8d267db5d4778e2f72afed9fa7d7b5R55-R56). If they're the same, whey not just pass them as props.
2. Is changing counties something that should involve TECH (then yes, ENV) or is it something that the content/data team will be handling on their own (then CMS).

Reply:
@kilemensi ahaa! Simple english now.. yes, let me add them on CMS